vitrina/redirections/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+
+
+class Redirections(models.Model):
+    objects = models.Manager()
+    created = models.DateTimeField(blank=True, null=True, auto_now_add=True)
+    name = models.TextField(blank=True, null=True)
```

Comment:
```suggestion
    name = models.CharField(max_length=255, blank=True)
```

We don't want to add extra burden of handling null values. Empty string is sufficient in our case.